src/main/webapp/js/httputil.js

```diff
@@ -31,4 +31,30 @@ function HttpUtil() {
         document.body.appendChild(form);
         form.submit();
     };
+
+    this.sendGetRedirect = function (destination, params) {
```

Comment:
This is a super weird way to make a GET "redirect". You can simply do `location.href = newUrl` and the browser will make a `GET` request to that url.

Reply:
To be more specific I _think_ this function can be replaced with this version:

```javascript
this.sendGetRedirect = function (destination, params) {
  var urlParams = Object.keys(params).map(function(k) { 
    encodeURIComponent(k) + '=' encodeURIComponent(params[k]) 
  }.join('&')
  location.href = destination + '?' + urlParams
}
```

(But please test it first, I just wrote it blind 😬 )

Reply:
At first the redirect was by POST and we changed it to GET. That code remain from the first logic but you are right. I will change it.

Reply:
By the way, my code snippet had a missing `return` which I just added (and it might have other issues too)

Reply:
I have use your function. I had to fix a little but the base works :).